meadControllerNodeWithWebApp/meadControllerNodeWithWebApp.ino

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
```
 //Start at 59 so that we print right away on boot
 int writeCounter = 59;
```
This is usually a code smell

I'd suggest you start at zero.

`if writeCounter == 0 {...}`
and at the end of loop
```
writeCounter++;
writeCounter %= 60;
```